deploy/lib/server_config.rb

```diff
@@ -672,16 +679,12 @@ def bootstrap
       assigns = ''
       internals = @properties['ml.system-dbs'].split ','
       internals.each do |db|
-        repnames = ''
-        repassigns = ''
-        (1..nr).each do |i|
-          repnames = repnames + %Q{
-                <replica-name>#{db}-rep#{i}</replica-name>}
-          repassigns = repassigns + %Q{
```

Comment:
If I'm reading this correctly, and I could be wrong here, are you only allowing one replica now?